tests/pages/login_page.py

```
@@ -0,0 +1,100 @@
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)